docs/packages/markdown-it.md

```diff
@@ -43,7 +43,7 @@ const highlighter = await getHighlighterCore({
   langs: [
     import('shikiji/langs/javascript.mjs'),
   ],
-  loadWasm: getWasmInlined
+  loadWasm: getWasm
```

Comment:
Totally up to you, but wouldn't using `loadWasm` as import name allow `loadWasm` shorthand syntax?

Reply:
The reason we have a different name here is that `loadWasm` is an exported function, and we could have usage like:

```ts
import { loadWasm } from 'shikiji/core'
import getWasm from 'shikiji/wasm'

await loadWasm(getWasm)
```